src/exporter/models/exporterStore.ts

```diff
@@ -10,11 +10,12 @@ import { Polygon } from '@turf/helpers';
 import { ApiHttpResponse } from '../../common/models/api-response';
 import { ResponseState } from '../../common/models/ResponseState';
 import { getLayerUrl } from '../../common/helpers/layer-url';
+// import MOCK_EXPORTED_PACKAGES from '../../__mocks-data__/exportedPackages';
 import { searchParams } from './search-params';
 import { IRootStore } from './rootStore';
-import { IGeoPackage } from './geoPackage';
+import { IExportTaskStatus } from './exportTaskStatus';
 
-export type GeoPackageResponse = IGeoPackage[];
+export type ExportTaskStatusResponse = IExportTaskStatus[];
```

Comment:
check eslint error Check warning on line 70 (result is not used)

Reply:
Yes, I know. Leave it on purpose. Please neglect this warning